pyproject.toml

```diff
@@ -41,6 +44,7 @@ pydantic = {version = "<2", optional = true}
 python-multipart = "^0.0.5"
 sanic = {version = ">=20.12.2,<22.0.0", optional = true}
 aiohttp = {version = "^3.7.4.post0", optional = true}
+uvloop = "^0.16.0"
```

Comment:
I see that the commit says "temp", but what's this for?

Reply:
Oh wait, this isn't uvicorn

Reply:
yeah, uvloop < 0.16 doesn't work on python 3.10, not sure why it doesn't automatically get installed